cmd/migrate_from_qdrant.go

```diff
@@ -125,7 +126,7 @@ func (r *MigrateFromQdrantCmd) Run(globals *Globals) error {
 		return fmt.Errorf("failed to connect to target: %w", err)
 	}
 
-	err = r.prepareMigrationOffsetsCollection(ctx, sourceClient)
+	err = r.prepareMigrationOffsetsCollection(ctx, targetClient)
```

Comment:
Store offsets in the target collection to reuse the implementation across various source providers.